### DISCUSSION TITLE
zie/ziewin/3.0/doc/qb/abt

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#73] [Q&A]
# zie/ziewin/3.0/doc/qb/abt

This book describes how to install, configure, and start HCL Z and I Emulator for Windows . After you get Z and I Emulator for Windows up and running and begin to perform various tasks, use the online ...

https://help.hcl-software.com/zie/ziewin/3.0/doc/qb/abt.html

**Comment by KunioMatsuo** [comment]
"Windows® refers to Microsoft® Windows® 7, Windows® 8/8.1, Windows® 10, Windows® Server 2008, and Windows® Server 2012" is ancient.  This should be updated.
